substrate/frame/system/src/extensions/weights.rs

```diff
@@ -68,74 +68,70 @@ impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
 	/// Proof: `System::BlockHash` (`max_values`: None, `max_size`: Some(44), added: 2519, mode: `MaxEncodedLen`)
 	fn check_genesis() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `54`
+		//  Measured:  `30`
 		//  Estimated: `3509`
-		// Minimum execution time: 3_876_000 picoseconds.
-		Weight::from_parts(4_160_000, 3509)
+		// Minimum execution time: 3_388_000 picoseconds.
+		Weight::from_parts(3_577_000, 3509)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 	}
 	/// Storage: `System::BlockHash` (r:1 w:0)
 	/// Proof: `System::BlockHash` (`max_values`: None, `max_size`: Some(44), added: 2519, mode: `MaxEncodedLen`)
 	fn check_mortality_mortal_transaction() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `92`
+		//  Measured:  `68`
 		//  Estimated: `3509`
-		// Minimum execution time: 6_296_000 picoseconds.
-		Weight::from_parts(6_523_000, 3509)
+		// Minimum execution time: 6_442_000 picoseconds.
+		Weight::from_parts(6_703_000, 3509)
```

Comment:
here we also remove the block hash read manually?

Reply:
Same as above.